glean-core/tests/common/mod.rs

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+pub fn tempdir() -> (tempfile::TempDir, String) {
+    let t = tempfile::tempdir().unwrap();
+    let name = t.path().display().to_string();
+    (t, name)
+}
+
+pub const GLOBAL_APPLICATION_ID: &str = "org.mozilla.glean.test.app";
```

Comment:
Creating the `glean` object is the same across all tests.
In `tests/metrics.rs` we have;

```rust
fn new_glean() -> (Glean, TempDir) {
    let dir = tempfile::tempdir().unwrap();
    let tmpname = dir.path().display().to_string();

    let glean = Glean::new(&tmpname, GLOBAL_APPLICATION_ID).unwrap();

    (glean, dir)
}
```

Can we use that everywhere instead of just the `tempdir` function?

Reply:
Sure, that's a good idea.  Do you think that belongs in this file or in `common/mod.rs`?

Reply:
If we re-use it in mod.rs.